shared/chat/conversation/messages/wrapper/shared.js

```diff
@@ -30,7 +30,7 @@ const colorForAuthor = (user: string, isYou: boolean, isFollowing: boolean, isBr
 
 const UserAvatar = ({author, showImage, onAuthorClick}) => (
   <Box style={styles.userAvatar}>
-    {showImage && <Avatar size={24} username={author} skipBackground={true} onClick={onAuthorClick} />}
```

Comment:
I've noticed this recently too.

Reply:
so the core issue was the send indicator actually. This exists on master but is much more subtle. if you click into a conversation with a bunch of multi single line messages you'll see it jumps up a tiny bit after the initial render.

couple of issues:
1. the state of the send indicator initializes to visible, but the on mount does setstate (usually) to visible false so it always renders once then goes away
1. when the avatar and the send are visible at the same time it actually makes the height pretty big (larger than one line). and after the fact if the send's internal state changes it doesn't actually re-measure (which is a violation of react virtualized). Instead of doing all that (even with remeasure you'd see it jump around) I changed it so send is always absolutely positioned and never grows the row height. This means if you send and its the first thing the send animation will be below the avatar. @cecileboucheron can chime in if this feels too bad, if so we have to change something (maybe it can bleed down and out of the container or something)

Reply:
also i refactored out some pieces so its be easier to debug in the react view instead of all these nested boxes